service/key/key.go

```diff
@@ -278,3 +278,7 @@ func IsCAPICluster(obj metav1.Object) bool {
 func IngressClassName() string {
 	return "nginx"
 }
+
+func OpsgenieKey() string {
+	return "opsgenie.key"
```

Comment:
Can't we just have ?

```suggestion
	return "opsgenie"
```

Reply:
We could, I named that because the value is mounted into a secret and we know what this field is about

Reply:
It feels redundant to call a key key to me.